### ISSUE TITLE
/content/devices/multi demo doesn't work

### ISSUE THREAD

**Issue by michaelstephendavies** [webrtc/samples#698]
In Chrome 46.0.2490.86 m I get the error

```
Error:  TypeError: alloutputSelectors.forEach is not a function(…)
```

The problem is [here](https://github.com/webrtc/samples/blob/gh-pages/src/content/devices/multi/js/main.js#L41). `document.querySelectorAll` returns a `NodeList`, not an array, and `NodeList`s do not have a `forEach` method.

**Comment by samdutton** [comment]
Thanks for reporting! 

I'll fix it later today.

**Comment by KaptenJansson** [comment]
Thanks for reporting, sloppy of me, I was sure it was working......

**Comment by KaptenJansson** [comment]
@samdutton I've fixed it in #699, PTAL.
